classes/ObjectModel.php

```diff
@@ -2206,7 +2206,28 @@ public static function createDatabase($className = null)
             if (isset($field['lang']) && $field['lang'] || isset($field['shop']) && $field['shop']) {
                 continue;
             }
-            $sql .= '`'.$fieldName.'` '.$field['db_type'];
+
```

Comment:
Can you go through the switch in case `$field['db_type']` is missing? It would otherwise limit the possibilities that modules do have right now.

Reply:
It is a great idea to add some defaults. Less typing to add the default types. Always nice :)